src/index.js

```diff
@@ -1 +1,4 @@
-import './main.css';
\ No newline at end of file
+import './main.css';
+import getData from './modules/dom.js';
+
+console.log(getData);
```

Comment:
Here, I think you forgot a console.log statement in the file. In my humble opinion, the console.log statement is not in tandem with JavaScript best practices.